saleor/account/forms.py

```diff
@@ -120,10 +119,73 @@ def get_users(self, email):
         return active_users
 
     def send_mail(
-            self, subject_template_name, email_template_name, context,
-            from_email, to_email, html_email_template_name=None):
+            self,
+            subject_template_name,
+            email_template_name,
+            context,
+            from_email,
+            to_email,
+            html_email_template_name=None):
         # Passing the user object to the Celery task throws an
         # error "'User' is not JSON serializable". Since it's not used in our
         # template, we remove it from the context.
         del context['user']
         emails.send_password_reset_email.delay(context, to_email)
+
+
+class EmailChangeForm(forms.ModelForm):
+    """ Allow changing email """
+
+    user_email = forms.EmailField(
+        label=pgettext("Label of the user's email field", "User Email"))
+
+    error_messages = {
+        'requests_exceeded':
+        pgettext_lazy(
```

Comment:
This should start on the previous line

Reply:
@Pacu2 so it should looks like this?
`        'same_email': pgettext_lazy(
            "Email can't be the same", 'New email address cannot be the '
            'same as your current email address'),`

I don't know wy my code formatter split this to new lines....

Reply:
Right, but with an extra indent in front of `Email can't`...